README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
```

Comment:
Great use of a title and subheaders using a variety of `#` to break up the flow of your document.